dietpi/dietpi-software

```diff
@@ -12759,6 +12805,47 @@ _EOF_
 
 		fi
 
+		software_id=191 # snapcast-client
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+			# Client IP needs to be added to allowed IP list
+			local snapcast_server_ip=
+			local snapcast_server_port=1704
+			local invalid_entry=
+			while :
+			do
+				if G_WHIP_INPUTBOX "${invalid_entry}Please enter the IP address of your snapcast server machine" && [[ $G_WHIP_RETURNED_VALUE =~ ^[0-9.]+$ ]]; then
+					snapcast_server_ip=$G_WHIP_RETURNED_VALUE
+					break
+				else
+					invalid_entry='[ERROR] Please enter a valid IP address. Please retry...\n\n'
```

Comment:
When interactive input is forced, set `aSOFTWARE_INTERACTIVE[$software_id]=1` in the init section above. This is important to avoid infinite loops when no input terminal is available, during automated firstrun installs and such.

Reply:
Thanks - done.
It does support it on the server but isn't very clear on the client - I guess it does as the server can be on an IPv6.